### DISCUSSION TITLE
PipePipe Subs and Channels

### DISCUSSION THREAD

**Discussion by chris02000** [InfinityLoop1308/PipePipe#426] [Q&A] [answered]
How do I add subscriptions to specific channels?

Also is it possible to export Bookmarked playlist, subscription etc to New pipe, should I need to in the future?

Many thanks
Chris

**Comment by chris02000** [comment]
Any advice appreciated.

**Reply by InfinityLoop1308** [reply]
Hi @chris02000, for channels you can just open a channel and click subscribe.
NewPipe don't want to support the migration from PipePipe, so you can't import the backup data in NewPipe directly. However with tools like [PipePipe2NewPipe](https://github.com/shreyas-a-s/pipepipe-to-newpipe/) you should be able to convert the database. 
Feel free to ask more questions if you still have any :)

**Reply by chris02000** [reply]
I am opening channels to and have set up Photo, Computing and I have 100 YouTubers to which I subscribe, but how do I catalogise each subscription to a channel?

Maybe I am just being thick(-:

**Reply by InfinityLoop1308** [reply, marked as answer]
I think you can just use channel groups. When creating a new group you can add as many channels as you want in the same time.
